boards/st/nucleo_n657x0_q/doc/index.rst

```diff
@@ -199,33 +199,36 @@ First, connect the NUCLEO-N657X0-Q to your host computer using the ST-Link USB p
          .. note::
             For flashing, before powering the board, set the boot pins in the following configuration:
 
-            * BOOT0: 0
-            * BOOT1: 1
+            * BOOT0: 1
+            * BOOT1: 0
```

Comment:
I'm not happy with this change. The previous information was right. The issue is the board user manual is not very explicit on which jumper position relates to BOOTx values 0 or 1.
Actually, the board PCB has a nice "1" printed for one position of each BOOTx jumper. It happens that this position relates to value "0" not "1" of related BOOTx. The rational seems to be that the other jumper position is "2" (not printed on PCD but mentioned in the board user manual), relating to value "1" of the related BOOTx.

All in one, I need to restore the previous values but find a better wording to guide users.